examples/src/utils/example-helpers.ts

```diff
@@ -203,3 +203,29 @@ export async function createSDKFromEnvironment(
   await client.authenticate();
   return client;
 }
+
+export const getCogniteClient = async ({
+  project,
+  environment,
+  overrideToken
+}: {
+  project: string | null;
+  environment: string | null;
+  overrideToken: string | null;
+}): Promise<CogniteClient> => {
+  if (project !== null) {
```

Comment:
Invert `if` to avoid nesting

Reply:
This was what I did initially, but I didn't like the throw at the end. I agree about the nesting and defensive programming approach, though, so I think I like this better after all. What do you think?

```typescript
  if (project === null) {
    return new CogniteClient({
      appId: 'reveal.example.example',
      project: 'dummy',
      getToken: async () => 'dummy'
    });
  }

  if (overrideToken !== null) {
    return createSDKFromToken('reveal.example.example', project, overrideToken);
  }

  if (environment !== null) {
    return await createSDKFromEnvironment('reveal.example.example', project, environment);
  }

  throw new Error('Unsupported arguments to getCogniteClient');
```

Reply:
Perhaps we can make the two latter conditions into `overrideToken !== null && project !== null` and `environment !== null && project !== null`? And then move the default SDK construction to the end? A little redundant, but I think I like it better

Reply:
Make sense. Updated in: [5caf7a8](https://github.com/cognitedata/reveal/pull/3594/commits/5caf7a8c57444988a674980d45f28c0cc9e97c2f)